modules/rosa-architecture.adoc

```diff
@@ -32,3 +32,15 @@ The following types of clusters are available with ROSA Classic:
 * Single zone cluster - The control plane and worker nodes are hosted on a single availability zone.
 
 * Multi-zone cluster - The control plane is hosted on three availability zones with an option to run worker nodes on one or three availability zones.
+
+Alternatively, you can install a ROSA Classic cluster using AWS PrivateLink, which is hosted on private subnets only.
+
+The following images depict how machine pools work within ROSA and {hcp-title} clusters:
```

Comment:
This statement is misplaced/wrong. We are here within ROSA Classic Architecture. Why is this image about HCP within ROSA Classic?